app/models/bops/neighbour_response.rb

```diff
@@ -10,15 +10,16 @@ def create(planning_application_id, data:)
           response: construct_response(data),
           address: data[:address],
           email: data[:email],
-          summary_tag: data[:summary_tag]
+          summary_tag: data[:summary_tag],
+          files: data[:files]
         )
       end
 
       def construct_response(params)
         tags = params[:tags].scan(/[a-z]+[_[a-z]]*/)
 
         tags.map do |tag|
-          "#{tag.to_s.humanize}: #{params[:"#{tag}"]}\n"
+          "#{tag.to_s.humanize}: #{params[:"#{tag.strip}"]}\n"
```

Comment:
Couldn't we just do:
```suggestion
          "#{tag.to_s.humanize}: #{params[tag.strip.to_sym]}\n"
```